Crane/app/src/main/java/androidx/compose/samples/crane/calendar/model/DaySelected.kt

```diff
@@ -17,14 +17,16 @@
 package androidx.compose.samples.crane.calendar.model
 
 import androidx.compose.samples.crane.data.CalendarYear
+import java.util.Locale
 
 data class DaySelected(val day: Int, val month: CalendarMonth, val year: CalendarYear) {
     val calendarDay = lazy {
         month.getDay(day)
     }
 
     override fun toString(): String {
-        return "${month.name.substring(0, 3).capitalize()} $day"
+        return "${month.name.substring(0, 3)
+            .replaceFirstChar { if (it.isLowerCase()) it.titlecase(Locale.getDefault()) else it.toString() }} $day"
```

Comment:
The expression is too complex to be contained within a string, prefer writing it to a variable first